content/en/docs/concepts/configuration/secret.md

```diff
@@ -726,6 +726,11 @@ The output is similar to:
 1f2d1e2e67df
 ```
 
+#### Environment variables are not updated after a secret update
+
+If a container already consumed a secret in an environment variable, a secret update will not be seen by the container unless it is restarted.
+There are third party solutions such as [Reloader](https://github.com/stakater/Reloader) for triggering restarts when secrets change.
```

Comment:
Looking at https://kubernetes.io/docs/contribute/style/content-guide/#third-party-content and

> Docs can link to third-party open source software (OSS) outside the Kubernetes project only if it's necessary for Kubernetes to function

then I think we can mention that third party solutions exist, but that we couldn't actually link to this specific one. That's a shame but I understand why.

Reply:
Indeed, thanks for checking this.
I have updated.